diff --git a/FBSimulatorControl/Commands/FBSimulatorAccessibilityCommands.m b/FBSimulatorControl/Commands/FBSimulatorAccessibilityCommands.m
index cb613a2..203de42 100644
--- a/FBSimulatorControl/Commands/FBSimulatorAccessibilityCommands.m
+++ b/FBSimulatorControl/Commands/FBSimulatorAccessibilityCommands.m
@@ -161,6 +161,7 @@ @implementation FBSimulatorAccessibilitySerializer
 @end
 
 static NSString *const DummyBridgeToken = @"FBSimulatorAccessibilityCommandsDummyBridgeToken";
+static const unsigned long long FBSimulatorAccessibilityXCTestClientType = 2;
 
 @interface FBSimulatorAccessibilityCommands_SimulatorBridge : NSObject <FBAccessibilityOperations>
 
@@ -410,6 +411,7 @@ - (AXPTranslationCallback)accessibilityTranslationDelegateBridgeCallbackWithToke
     };
   }
   return ^ AXPTranslatorResponse * (AXPTranslatorRequest *request){
+    request.clientType = FBSimulatorAccessibilityXCTestClientType;
     [simulator.logger logFormat:@"Sending Accessibility Request %@", request];
     dispatch_group_t group = dispatch_group_create();
     dispatch_group_enter(group);
