diff --git a/Configuration/Shared.xcconfig b/Configuration/Shared.xcconfig
index d0746b8e..f4dfe6ce 100644
--- a/Configuration/Shared.xcconfig
+++ b/Configuration/Shared.xcconfig
@@ -51,6 +51,6 @@ GCC_WARN_UNUSED_VARIABLE = YES
 
 // SDK & OS Versions
-MACOSX_DEPLOYMENT_TARGET = 10.14
+MACOSX_DEPLOYMENT_TARGET = 12.0
 SDKROOT = macosx
 
 // Compile-Time Settings
diff --git a/Shims/Shimulator/Configs/Maculator.xcconfig b/Shims/Shimulator/Configs/Maculator.xcconfig
index 8cccb535..e4da19b4 100644
--- a/Shims/Shimulator/Configs/Maculator.xcconfig
+++ b/Shims/Shimulator/Configs/Maculator.xcconfig
@@ -6,6 +6,6 @@ DYLIB_COMPATIBILITY_VERSION = 1
 PRODUCT_NAME = Maculator
 EXECUTABLE_PREFIX = lib
 SDKROOT = macosx
-MACOSX_DEPLOYMENT_TARGET = 10.13
+MACOSX_DEPLOYMENT_TARGET = 12.0
 COPY_PHASE_STRIP = NO
 CODE_SIGN_IDENTITY = - // Maculator Shims must be CodeSigned, to work for injection.
diff --git a/Shims/Shimulator/Configs/Shimulator.xcconfig b/Shims/Shimulator/Configs/Shimulator.xcconfig
index 442e69fe..c3ecee38 100644
--- a/Shims/Shimulator/Configs/Shimulator.xcconfig
+++ b/Shims/Shimulator/Configs/Shimulator.xcconfig
@@ -6,6 +6,6 @@ DYLIB_COMPATIBILITY_VERSION = 1
 PRODUCT_NAME = Shimulator
 EXECUTABLE_PREFIX = lib
 SDKROOT = iphonesimulator
-IPHONEOS_DEPLOYMENT_TARGET = 11.0
+IPHONEOS_DEPLOYMENT_TARGET = 15.0
 COPY_PHASE_STRIP = NO
 CODE_SIGN_IDENTITY = -  // Simulator Shims must be CodeSigned, to work for injection.
